tests/cases/AbstractMenuItemsContainerTest.php

```diff
@@ -140,7 +146,10 @@ public function testVisibleItemsOn(): void
 
 		$container = $this->createPartialMockAbstractMenuItemsContainer(function (
 			MockInterface $container
-		) use ($itemA, $itemB): void {
+		) use (
+			$itemA,
+			$itemB
+): void {
```

Comment:
Also bad indentation

Reply:
I don't know why phpcsfix did that. :open_mouth: